test/system/045-start.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats   -*- bats -*-
+
+load helpers
+
+@test "podman start --all - start all containers" {
+    run_podman run -d $IMAGE /bin/true
+    cid_none="$output"
+    run_podman run -d --restart=always $IMAGE /bin/true
+    cid_always="$output"
+    run_podman start --all
+    is "$output" "$cid_none"
+    #[[ "$output" == *"$cid_none"* ]]
+    #[[ "$output" == *"$cid_always"* ]]
+}
+
+@test "podman start --all with incompatible options" {
+    expected="Error: either start all containers or the container(s) provided in the arguments"
+    run_podman 125 start --all 12333
+    cid_none="$output"
```

Comment:
I think you meant to write `is "$output" "$expected" "start --all, with args, throws error"`